### DISCUSSION TITLE
X

### DISCUSSION THREAD

**Discussion by P4KM4N** [twbs/bootstrap#38986] [General] [answered]
I was asking myself what's the future of Bootstrap. I know it's open source and it have a gigantic community, but it all started under Twitter umbrella, now that Elon is in charge, mdo and fat working on another project.

Everything I see these days are build with Tailwind CSS. Since 2023 Tailwind have take the lead.
https://npmtrends.com/bootstrap-vs-tailwindcss

I love Bootstrap, since V1. Every websites I built since the last 10 years was built with it.
But I wonder if this is the end of an era.

**Comment by julien-deramond** [comment, marked as answer]
> I know it's open source and it have a gigantic community, but it all started under Twitter umbrella, now that Elon is in charge, mdo and fat working on another project.

Bootstrap has been independent of Twitter for some years now and mdo is still working on the project in his spare time, like all the maintainers of Bootstrap.

> Everything I see these days are build with Tailwind CSS. Since 2023 Tailwind have take the lead.

Regarding Tailwind, both have a different approach and are different tools so folks use the tools they think are the most appropriate to their needs. A lot of companies still use Bootstrap.

> But I wonder if this is the end of an era.

IDK but we're still putting effort into maintaining Bootstrap.

**Reply by P4KM4N** [reply]
Merci Julien,

Thanks for calming me down and give me trust in the future of the project.
We need to double down our effort all over the Web to push BS! 

Exemple: 
Creating starter kits for Statamic
https://statamic.com/starter-kits

**Reply by dithom** [reply]
I was asking myself exactly the same thing! Thanks @P4KM4N for the question and thanks @julien-deramond for the calming answer. I did not know, that Bootstrap is independent of Twitter/X.
